src/SignalRServiceExtension/Bindings/EndpointConnectionInfo.cs

```diff
@@ -3,10 +3,11 @@
 
 using Microsoft.Azure.SignalR;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
 
 namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
 {
-    [JsonObject]
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
```

Comment:
why `CamelCaseNamingStrategy` is required? the default one doesn't work?

Reply:
Newtonsoft is not camel case by default?